indico/modules/events/contributions/templates/display/contribution_display.html

```diff
@@ -2,6 +2,7 @@
 
 {% from 'events/display/common/_conferences.html' import render_attachments %}
 {% from 'events/display/indico/_common.html' import render_location, render_users, render_speakers %}
+{% from 'events/management/_affiliation.html' import render_affiliation %}
```

Comment:
Maybe we should move it outside that `management` subdir if we start using it outside management?